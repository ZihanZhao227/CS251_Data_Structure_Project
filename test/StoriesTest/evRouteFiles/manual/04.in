20
0 0 0 CITY Start
1 25 103 RESTAURANT RESTAURANT_1
2 141 180 CHARGING_STATION CHARGING_STATION_2
3 187 67 RESTAURANT RESTAURANT_3
4 168 43 CHARGING_STATION CHARGING_STATION_4
5 109 16 CHARGING_STATION CHARGING_STATION_5
6 90 193 REGULAR_WAYPOINT REGULAR_WAYPOINT_6
7 70 164 CHARGING_STATION CHARGING_STATION_7
8 31 46 CHARGING_STATION CHARGING_STATION_8
9 119 6 REGULAR_WAYPOINT REGULAR_WAYPOINT_9
10 162 47 CHARGING_STATION CHARGING_STATION_10
11 187 118 RESTAURANT RESTAURANT_11
12 12 28 RESTAURANT RESTAURANT_12
13 59 68 CHARGING_STATION CHARGING_STATION_13
14 184 191 CHARGING_STATION CHARGING_STATION_14
15 173 45 CHARGING_STATION CHARGING_STATION_15
16 81 187 CHARGING_STATION CHARGING_STATION_16
17 102 79 TOURIST_ATTRACTION TOURIST_ATTRACTION_17
18 55 31 CHARGING_STATION CHARGING_STATION_18
19 200 200 CITY End
