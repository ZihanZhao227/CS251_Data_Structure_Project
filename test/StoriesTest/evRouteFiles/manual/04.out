0 19 50.000 -1.0
NO_PATH
